src/components/CRUDDash.tsx

```diff
@@ -23,6 +23,16 @@ const CRUDDash: React.FC<CrudProps> = ({ firestore, data }) => {
   const [modifyEntitySecondProperty, setModifyEntitySecondProperty] = useState('')
   const [entityType, setEntityType] = useState<'Categories' | 'Types'>('Categories')
 
+  const handleDownload = () => {
```

Comment:
Unfortunately, it seems like this solution doesn't work in Chrome or Firefox. I added a content-disposition header in cache_script.py that may fix the issue, but we'll have to merge to test it. **Before that, you should remove this function and replace the <button> element that calls it with a normal <a> link instead.**